app/javascript/hyacinth_ui_v1/components/digital_objects/children/Children.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
-import produce from 'immer';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
 
 import digitalObjectInterface from '../digitalObjectInterface';
+import AssetNew from '../new/AssetNew';
 import TabHeading from '../../ui/tabs/TabHeading';
+import GraphQLErrors from '../../ui/GraphQLErrors';
 
-class Children extends React.Component {
-  // state = {
-  //   digitalObject: {},
-  // }
-
-  // componentDidMount() {
-  //   const { match: { params: { id } } } = this.props;
-  //
-  //   digitalObject.get(id)
-  //     .then((res) => {
-  //       console.log(res.data);
-  //       this.setState(produce((draft) => {
-  //         draft.digitalObject = res.data.digitalObject;
-  //       }));
-  //     });
-  // }
-
-  render() {
-    const { data: { uid } } = this.props;
-
-    return (
-      <>
-        <TabHeading>Manage Child Assets</TabHeading>
-        <p>Child Digital Objects</p>
-        <p># Total</p>
-
-
-      </>
-    );
+const getChildAssetsQuery = gql`
+  query ChildAssets($id: ID!){
+    childAssets(id: $id) {
+      id
+    }
   }
-}
+`;
+
+const Children = (props) => {
+  const { match: { params: { id } } } = props;
```

Comment:
Since this is a functional component, I think you could use the useParams hook here instead.  If you search through the project code for `useParams`, you'll see some examples.

Reply:
params matching is totally unnecessary, so I removed it in favor of a simple destructuring assignment.

Reply:
Great, thanks!